### PR TITLE
Display all review sections in responsePage

### PR DESCRIPTION
**Pull request by Vilmo18**
Fix problem of method `renderReviewResponse()` and improve `renderCompleteReviews()`
Add a new method `getTotalScore()` for getting total review

Correct of minor bugs in:
- `reviewForm` page for computing totalscore during reviewing
- `responseList` page for displaying email instead of language

### CODE REVIEW

File: webapp/src/pages/ResponsePage/ResponsePage.js [hunk 1]
```diff
@@ -114,42 +143,57 @@ class ResponsePage extends Component {
...
+                        </h4>
+                        {this.state.reviewForm.review_sections.map(section => (
+                            <div key={section.id} className="section">
+                            <div className="flex baseline"><h3>{section.headline}</h3></div>
+                                {this.renderReviewResponse(val, section)}
+                            </div>
+                        ))}
+                        <hr/>
+                        <div className="review-total-score">
+                        <h5>Total Score: {this.state.totalScore}</h5>
```
Comment (avishkar58) [inline webapp/src/pages/ResponsePage/ResponsePage.js:184]: I think this should be the total score of the review that is being rendered, no?

**Review by avishkar58 — CHANGES_REQUESTED** [review]
Thank you very much for the contribution Yvan! :) Have left some pointers where i think the naming of some of our services is confusing and doesn't quite correspond to what it actually does.

File: webapp/src/pages/ResponsePage/ResponsePage.js (hunk 1 above)
Comment (Vilmo18) [reply webapp/src/pages/ResponsePage/ResponsePage.js:184]: Indeed

**Comment by Vilmo18** [comment]
I have fixed it.

**Pull request closed without merging**